lib/ansible/modules/cloud/google/gcp_compute_route.py

```diff
@@ -267,10 +267,10 @@ def main():
     if fetch:
         if state == 'present':
             if is_different(module, fetch):
-                fetch = update(module, self_link(module), kind, fetch)
+                fetch = update(module, self_link(module), kind)
```

Comment:
Looks like I forgot a rebase somewhere down the line. This is the result from that.